Ports/llvm/patches/llvm.patch

```diff
@@ -0,0 +1,178 @@
+diff -ruN llvm-orig/llvm-project-llvmorg-12.0.0/llvm/include/llvm/Support/ConvertUTF.h llvm-project-llvmorg-12.0.0/llvm/include/llvm/Support/ConvertUTF.h
+--- llvm-orig/llvm-project-llvmorg-12.0.0/llvm/include/llvm/Support/ConvertUTF.h	2021-04-06 19:38:18.000000000 +0300
++++ llvm-project-llvmorg-12.0.0/llvm/include/llvm/Support/ConvertUTF.h	2021-06-09 16:04:10.069288846 +0300
+@@ -200,19 +200,19 @@
+ * Converts a UTF-8 StringRef to a std::wstring.
+ * \return true on success.
+ */
+-bool ConvertUTF8toWide(llvm::StringRef Source, std::wstring &Result);
++bool ConvertUTF8toWide(llvm::StringRef Source, std::string &Result);
```

Comment:
This is water under the bridge now, but it would've been nice if the patches were separated by topic,
e.g. "remove wstring", "insert `ifdef __serenity__` where appropriate", etc.
This would help us not re-patch everything when something changes and makes a part of a patch redundant.